### PR TITLE
Update versions for 2.0.0

### PR DESCRIPTION
**Pull request by samtstern**
Leaving this PR open until we get:

  * Support lib `28.0.0`
  * AGP `3.2.0`

This is just something to remind me what I am waiting for.

### CODE REVIEW

**Comment by samtstern** [comment]
Nice!  Can jump on this on Monday then.

**Comment by chintansoni202** [comment]
Hey @samtstern I'm super excited to use AndroidX libraries in production along with EasyPermissions. Do you have any tentative date by which we would be able to use EasyPermissions with AndroidX ?

**Comment by XinyueZ** [comment]
@chintansoni202 According to the commitments there's still everything under support v4/v7 namespace.
I have a temp workaround fork which is in **androidx** when you like you can use it for temp until the official version is released.
https://github.com/XinyueZ/easypermissions

File: build.gradle [hunk 1]
```diff
@@ -34,7 +34,7 @@ ext {
 
     bintrayOrg = 'easygoogle'
 
-    support_library_version = '28.0.0-rc01'
+    support_library_version = '28.0.0-rc02'
```
[the remark this answers is not in the document]
Comment (samtstern) [reply build.gradle]: I will do that in a follow-up pull request!

**Pull request merged**